installer/mac/ChainCore/Config.swift

```diff
@@ -7,30 +7,27 @@ class Config: NSObject {
 
     /// Checks if the port is in use by another process.
     static func portInUse(_ port: UInt) -> Bool {
-        let sock = socket(PF_INET, SOCK_STREAM, IPPROTO_TCP)
-        if sock <= 0 {
-            return false
-        }
+        var output : [String] = []
+
+        let task = Process()
+        task.launchPath = "/usr/sbin/lsof"
```

Comment:
It would make sense, but we need a consistent solution for Postgres as well. I haven't investigated how `pg_isready` would respond to this.

Another thing to understand in the future is why this code (previously used [in the Postgres.app launcher](https://github.com/PostgresApp/PostgresApp/blob/cd8016c834a4fec26b6e4a449b6428eed9e00bc6/Postgres/Server.swift#L311-L337)) doesn't function appropriately here.